sdks/python/apache_beam/ml/inference/pytorch_inference.py

```diff
@@ -188,20 +216,24 @@ def __init__(
         Otherwise, it will be CPU.
     """
     self._state_dict_path = state_dict_path
-    if device == 'GPU' and torch.cuda.is_available():
+    if device == 'GPU':
+      logging.info("Device is set to CUDA")
```

Comment:
I wonder if some of this logging is getting to be a little too much.  Were you using some of it to debug?

For example you have "device is set to" xxx in a couple spots. Is that still important logging to keep?

Reply:
I wanted to be super explicit about the detection of accelerators so users wouldn't have to second guess the type of environment they were operating in. (due to the nature of having multiple workers in DataFlow, I can imagine there would be future scenarios in which we have some GPU inconsistencies across the workers)